### ISSUE TITLE
Reading 'Group' between the text boxes while navigating in scan mode. 

### ISSUE THREAD

**Issue by JasonAment** [microsoft/AdaptiveCards#2239]
Please reference this bug for repro steps and videos
https://github.com/Microsoft/BotFramework-WebChat/issues/1349

**Comment by shalinijoshi19** [comment]
@shalinijoshi19 to follow up see if there is a dup in ADO that @dclaux had triaged earlier last week.

**Comment by RebeccaAnne** [comment]
We believe this should be fixed, please reactivate if you are still seeing this issue.
